web/src/linkifiers.ts

```diff
@@ -73,20 +83,28 @@ function python_to_js_linkifier(pattern: string, url: string): [RegExp | null, s
             throw error;
         }
     }
-    return [final_regex, url];
+    const url_template = url_template_lib.parse(url);
+    blueslip.info(`Linkifier info ${String(final_regex)} ${url}`, group_number_to_name);
```

Comment:
Did we leave in this debug message intentionally?

Reply:
No. This needs to be removed.

Reply:
#25822